src/layout/layout.scss

```diff
@@ -37,7 +37,7 @@
   }
 
   .Layout-main {
-    grid-column: 2 / span 2;
+    grid-column: math.div(2, span) 2;
```

Comment:
As @simurai pointed out, this is not a division, but a [CSS grid syntax](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-column). Same with a couple of other lines below :)

Reply:
🤦🏻 now I get it....

Reply:
brutally force pushed now... things never happened. :D But I still don't know how to fix the $variable situation.

Reply:
Ahhh gosh... today's really not my day. Don't say anything... saw it my self.